source/routing/raptor.cpp

```diff
@@ -221,17 +227,44 @@ RAPTOR::compute_all(const vec_stop_point_duration& departures_,
                     const std::vector<std::string>& forbidden_uri,
                     bool clockwise) {
 
-    std::vector<Path> result;
+    auto solutions = Solutions(Dominates(clockwise));
 
-    auto calc_dep = clockwise ? departures_ : destinations;
-    auto calc_dest = clockwise ? destinations : departures_;
+    const auto& calc_dep = clockwise ? departures_ : destinations;
+    const auto& calc_dest = clockwise ? destinations : departures_;
 
     first_raptor_loop(calc_dep, calc_dest, departure_datetime, disruption_active, allow_odt,
                       bound, max_transfers, accessibilite_params, forbidden_uri, clockwise);
 
-    const auto reader_results = read_solutions(*this, clockwise, departures_, destinations, disruption_active, accessibilite_params);
+    swap(labels, first_pass_labels);
+    auto best_labels_pts_for_snd_pass = snd_pass_best_labels(clockwise, best_labels_transfers);
+    init_best_pts_snd_pass(calc_dep, departure_datetime, clockwise, best_labels_pts_for_snd_pass);
+    auto best_labels_transfers_for_snd_pass = snd_pass_best_labels(clockwise, best_labels_pts);
+
+    const unsigned first_pass_count = count;
```

Comment:
I think it would be better to put that in a function for clarity.

It would also be nice to comment the general aim of this 2nd pass